rclcpp/test/benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+find_package(performance_test_fixture REQUIRED)
+
+# Give cppcheck hints about macro definitions coming from outside this package
+get_target_property(
+  ament_cmake_cppcheck_ADDITIONAL_INCLUDE_DIRS
+  performance_test_fixture::performance_test_fixture
+  INTERFACE_INCLUDE_DIRECTORIES)
```

Comment:
Is this necessary or working? I always assumed it needed to be defined before `ament_cmake_cppcheck` was processed in the CMake.

Reply:
Removing it did not seem to impact cppcheck in any way.